cmake/dependencies/openPMD.cmake

```
@@ -87,7 +87,7 @@ if(WarpX_OPENPMD)
     set(WarpX_openpmd_repo "https://github.com/openPMD/openPMD-api.git"
```

trying https://github.com/openPMD/openPMD-api/pull/1824
```suggestion
    set(WarpX_openpmd_repo "https://github.com/franzpoeschel/openPMD-api.git"
```